CHANGELOG.latest.md

```diff
@@ -1,2 +1,4 @@
-- Obfuscates calls to `AppTrackingTransparency` to prevent unnecessary rejections for kids apps when the framework isn't used at all. 
-    https://github.com/RevenueCat/purchases-ios/pull/486
+- Exposes `ownershipType` in `EntitlementInfo`, which can be used to determine whether a given entitlement was shared by a family member or purchased directly by the user. 
+    https://github.com/RevenueCat/purchases-ios/pull/483
+- Adds new `RCConfigurationError` type, which will be thrown when SDK configuration errors are detected.
```

Comment:
👍  for a second was wondering why this PR didn't contain the changes themselves, but this PR is more just to kick of the release from develop, right?

@vegaro there weren't any amazon-related changes in the ios repo to avoid like we did on android, were there?

Reply:
that's correct, this is just to kick off a build from develop. 

And Amazon store support doesn't affect purchases-ios, it's Android-only

Reply:
🤦‍♀️  of course